varnish/ci/Dockerfile_varnish5

```diff
@@ -0,0 +1,11 @@
+from debian:stretch
```

Comment:
You might want to move this to our docker library: https://github.com/DataDog/docker-library
Having it prebuilt would make the tests run quicker